assets/javascripts/discourse/templates/list/topic-list-title.raw.hbs

```diff
@@ -24,6 +24,11 @@
   </span>
 
   {{#if tilesOrSocial}}
-    {{raw "list/topic-users" topic=topic posterNames=posterNames}}
+    {{#if tilesStyle}}
+    </div>
```

Comment:
@merefield Likewise here. There shouldn't be just part of html element in a conditional; only in exceptional cases. Also please make sure you adhere to the formatting convention of one step of indentation for the contents of a conditional, as it makes the code easier to read.

Reply:
I've reverted this file to the current HEAD. All layout changes are now done with jQuery. A little worried about performance now though.